src/types/index.ts

```diff
@@ -7,3 +7,4 @@ export type WireDataListener<T> = (value: WireDataValue<T>) => Promise<void> | v
 export type WireDataGetter<T> = (that: IWireData<T>) => any;
 export type WireDataOnRemove = (key: string) => boolean;
 export type WireDataOnReset = (that: string, param: any) => any;
+export type WireDataListenersExecutionMode = 'sequential' | 'parallel';
```

Comment:
Remove use of constant strings infavor to use object keys and change definition of that type to enum, update all the places  where it is used.